cli/src/pcluster/resources/login_node/user_data.sh

```diff
@@ -89,7 +89,9 @@ write_files:
           "scheduler": "${Scheduler}",
           "stack_name": "${AWS::StackName}",
           "stack_arn": "${AWS::StackId}",
-          "use_private_hostname": "${UsePrivateHostname}"
+          "use_private_hostname": "${UsePrivateHostname}",
+          "auto_scaling_group_name": "${AutoScalingGroupName}",
+          "lifecycle_hook_name": "${LifecycleHookName}"
```

Comment:
we have two different lifecycle hook for this ASG: one on start and one for terminate. Better to be more exlicit.